js/pages/characterizations/components/feature-analyses/feature-analysis-view-edit.less

```diff
@@ -110,4 +110,15 @@
     font-weight: 500;
     margin-bottom: 1rem;
   }
+}
+
+
+.fe-design-disabled {
+  pointer-events: none;
+  opacity: 0.7;
+
+  .btn.btn-primary {
```

Comment:
Could you describe what is the goal of this construction? Not sure CSS selector used in a right way

Reply:
it simulates the css style 'disabled', if I make the button just more transparent its color is too light